keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -438,19 +450,20 @@ Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
 checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
+By checking the `kube_statefulset_status_replicas_available` metric.
```

Comment:
So if the metric exists, the feature is in use? Please specify what the user needs to check.